resources/i18n/qqq.json

```diff
@@ -39,16 +41,14 @@
 	"videos": "{{identical|Video}}",
 	"yes": "{{identical|yes}}",
 	"no": "{{identical|no}}",
-	"download-dir-missing": "Error text displayed when it turns out that the download directory doesn't exist.",
-	"download-dir-not-writable": "Error text displayed when files cannot be created/saved in the download directory due to permissions",
-	"add-bookmark": "Represents the action of adding a page as a bookmark",
-	"remove-bookmark": "Represents the action of removing a page from the bookmarks",
-	"open-link-in-web-browser": "Hint for the action of opening the link in a web browser",
+	"open-link-in-web-browser": "Ouvrir le lien dans le navigateur",
```

Comment:
This change must be undone (it was fixed via `git` in #1092 but it looks like translatewiki restored the previous value).